proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,536 @@
+   * a) Serious violations of regulations in the Articles of Association,
+     the purpose and aim of the Association or the Code of Conduct.
+   * b) Non-paiment of membership fees over a period of 12 months.
+   Members must be given the opportunity to justify their actions
+   before expulsion.
+6. Members leaving the Association or if the Association is liquidated do not
+   have a right to reimbursement for contributed assets.
+
+## 5. Organs of the Association
+
+* The General Assembly
+* The Board
+
+## 6. General Assembly
+
+1. The General Assembly consists of all core members, each with one vote.
+2. It takes place at least once a year, with invitations including a
+   preliminary agenda is sent to all core and extraordinary members at
+   least six weeks prior. The members may submit additional agenda items to
+   the board until two weeks before the assembly. The Board will publish the
+   final agenda online.
+3. An extraordinary General Assembly may be called if requested by at least
+   20 % of core members.
+4. The General Assembly has a quorum if the invitation was submitted in proper
+   form. Should the number of pro and contra votes should be equal, an
+   application is considered as dismissed.
+5. A 3/4 majority of votes is required to amend the Articles of Association or
+   decide the liquidation of the Association.
+6. Core members unable to attend may be represented by another member with
+   written authorization.
+7. Participation and voting in the General Assembly must also be possible via
+   electronic communication. By resolution of the Board, a General Assembly
+   may be held exclusively by these means.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is
+   responsible for all duties unless otherwise assigned.
+2. It elects the Board from a number of core members and can deselect board
+   members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges
+   the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of
+   the Association.
+6. Namely the annual accounts and the annual report have to be presented in
+   written format to the General Assembly for decision making and discharge
+   of the board. It calls on an auditor, who belong neither to the board nor
+   any other panel appointed by the board and who is not employed by the
+   association, in order to audit the accounting and annual report and
+   relate their results to the General Assembly. The auditors have access
+   to all of the Association’s accounting documents.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
+   elected for a 3-year term.
+
+   TODO: Define the role responsibilities.
+
```

Comment:
Chairman
organisation 
- control values / goals of organisation are honoured 
- organizing meetings
- organizing timeline / schedule
- events?
- represent organisation 
- official contacts/engagements fi GSOC
- writing 'status reports'
- organize onMixxx things
- organize offMixxx things
- analog postadress

member functions : 
- welcome new contributors / members
- contact with users (not helpdesk) organizing a newsletter / ezine
- make website 'live', motivate people to write posts
- organizing newsletter / ezine towards the members/users
- help possible discusions being solved
- keep good atmosphere
- intermediate between ...

Vice-Chairman
- tasks from chairman when chairman's absent
- delegetaed tasks from chairman

Reply:
Cool thank you this a good start. We need to take care not to overload the Board with task. I think we need also consider the situation where the Mixxx e.V. is running on pilot flame and the Board takes care that it does not goes out. The rest can be put into the by-law.